transport_nantes/mobilito/static/mobilito/js/geolocation_map.js

```diff
@@ -110,17 +110,36 @@ function onMapClick(e) {
 }
 map.on('click', onMapClick);
 
+// From django docs
```

Comment:
This is the third instance of this function to be introduced in our code base.  It should not be: we should realise from the second instance that it is useful (really, the library doesn't have a get cookie function?) and put it somewhere common.

Reply:
How is it different from `request.COOKIE`?